src/entries/demo/Demo.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { parse as parseQuery } from 'qs'
+import OpenApiRenderer from '../OpenApiRenderer'
+import './styles.scss'
+
+const definitionUrl = parseQuery(location.search.split('?')[1]).url
+
+export class Demo extends React.PureComponent {
+  state = {
+    definitionUrl
+  }
+
+  useUrlInput = () => {
+    this.setState({ definitionUrl: this.urlInput.value })
+  }
+
+  setUrlInput = (input) => { this.urlInput = input }
+
+  render () {
+    const { definitionUrl } = this.state
+
+    return (
+      <div className='oapi-demo'>
+        <div className='oapi-header'>
+          <button onClick={this.useUrlInput}>READ</button>
```

Comment:
Can we flip this? Counter intuitive to move "backwards" to hit read once you've added a URL.